ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -165,23 +200,15 @@ StackLayout {
         MessageContextMenu {
             id: messageContextMenu
         }
- 
+
         Connections {
             target: chatsModel
             onActiveChannelChanged: {
-                chatInput.textInput.forceActiveFocus(Qt.MouseFocusReason)
-                chatInput.suggestionsList.clear()
-                const len = chatsModel.suggestionList.rowCount()
-                for (let i = 0; i < len; i++) {
-                    chatInput.suggestionsList.append({
-                                           alias: chatsModel.suggestionList.rowData(i, "alias"),
-                                           ensName: chatsModel.suggestionList.rowData(i, "ensName"),
-                                           address: chatsModel.suggestionList.rowData(i, "address"),
-                                           identicon: chatsModel.suggestionList.rowData(i, "identicon"),
-                                           ensVerified: chatsModel.suggestionList.rowData(i, "ensVerified"),
-                                           localNickname: chatsModel.suggestionList.rowData(i, "localNickname")
-                                       });
-                }
+                chatInput.suggestions.hide();
+                populateSuggestions();
+            }
+            onMessagePushed: {
+                populateSuggestions();
```

Comment:
It would be preferable to only parse the new message in that case. Parsing over all the messages in the list for all new messages can be quite heavy in big channels I think.
For this signal, do we have access to the new message's id or something?

Reply:
in last commit the signal now returns the message index